packages/build/src/steps/run_step.ts

```diff
@@ -79,9 +79,15 @@ export const runStep = async function ({
     'build.execution.step.origin': origin,
     'build.execution.step.event': event,
   }
+
+  if (pluginPackageJson?.name && pluginPackageJson?.version) {
+    attributes['build.execution.step.plugin_name'] = pluginPackageJson.name
+    attributes['build.execution.step.plugin_version'] = pluginPackageJson.version
```

Comment:
@orinokai I'm going to set now the plugin version and name for all executions not only the failed ones

Reply:
great news!

Reply:
Just a thought here - maybe this should be extracted to separate PR that could be shipped faster than this whole PR (given the potential issues with `@opentelemetry/api` version), so that we could at least start monitoring duration of `onBuild` etc steps done by Next runtime?

Reply:
We already have `'build.execution.step.package_name': packageName,` above, should we just keep one instead to avoid confusion?